Documentation/codereview-policy.md

```diff
@@ -0,0 +1,30 @@
+### Code Review Policy
+
+#### How to code review pull requests  
+
+This is a guide for core developers and contributors who want to help with code reviewing PRs on the fullnode.  
+We welcome anyone to help with code reviews, from typos to suggestions in design.   
+
+#### Why are code reviews important?  
+It's essential that PRs are properly code-reviewed,   
+changes to the code, even small changes as incrementing an integer, can result in critical consensus bugs and lose of funds if not reviewed properly.  
```

Comment:
That example would fail a test, so I think that is bad example and represents that we have a dangerous codebase where you can increment an integer and lose funds.

Reply:
Again, in a perfect world yes, but we don't have hundred percent test coverage, so in practice it might not fail a test.

Reply:
I think this is good that devs feel its dangerous to make changes, they will be extra careful.
And its very hard to cover all aspects to the code.